Rentit-frontend/src/pages/userDetails/productCard.jsx

```diff
@@ -0,0 +1,64 @@
+import { useNavigate } from 'react-router-dom';
+import { useState } from 'react';
+import { CgRemove } from 'react-icons/cg';
+import { RiDeleteBinLine } from 'react-icons/ri';
+import axios from 'axios';
+// import { MdOutlineRemoveCircle } from 'react-icons/md';
+
+const ProductCard = (props) => {
+  const navigate = useNavigate();
+  const [data, setData] = useState(props.data);
+  const [count, setCount] = useState(props.count);
+  // const data = props.data;
+  const type = props.type;
+  const user = JSON.parse(sessionStorage.getItem('userDetails')).email;
+  const url = 'http://127.0.0.1:2000/api-rentit/v1/items';
```

Comment:
Add `http://127.0.0.1:2000` in `.env` and use `process.env` here.

Reply:
It is will be added in upcoming commits